coalib/coala_main.py

```diff
@@ -52,6 +54,10 @@ def run_coala(log_printer=None,
     :return:                        A dictionary containing a list of results
                                     for all analyzed sections as key.
     """
+    # Configure logging
+    with open('logging_config.json') as fd:
+        logging.config.dictConfig(json.load(fd))
```

Comment:
I'll change that.

As for the docs, what exactly should go there. It's just a file with the configurations, nothing fancy. Or you're saying documentation on how to use logging, i. e. names of the loggers to use?